packages/commonwealth/server/migrations/20230503183808-chain-timestamps.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      // Add altering commands here.
+      await queryInterface.addColumn(
+        'Chains',
+        'created_at',
+        {
+          type: Sequelize.DATE,
+          allowNull: true,
+        },
+        { transaction }
+      );
+
+      await queryInterface.addColumn(
+        'Chains',
+        'updated_at',
+        {
+          type: Sequelize.DATE,
+          allowNull: true,
+        },
+        { transaction }
+      );
+
+      await queryInterface.addColumn(
+        'Chains',
+        'deleted_at',
+        {
+          type: Sequelize.DATE,
+          allowNull: true,
+        },
+        { transaction }
+      );
+
+      const updateQuery = `
```

Comment:
What will the query do if no addresses found?

Reply:
It will just not set the value of created_at. When I ran this locally, that was mostly occurring for junk chains i.e. "test-1" "alex-test" etc.